doc/static/css/nrf.css

```diff
@@ -121,3 +121,43 @@
 div.numbered-step>h2::before, div.numbered-step>h3::before, div.numbered-step>h4::before, div.numbered-step>h5::before {
   background: #00a9ce;
 }
+
+/* styling for known issues page */
+.versiontag {
+ border: 1px solid #e97c25;
+ padding: 2px;
+ font-size: smaller;
+ color: #e97c25;
+ margin: 0px 5px;
+}
+
+#known-issues dl dt {
+ color: #000;
+ border-top: none;
+ background: transparent;
+}
+
+#versions-select {
+ display: inline-block;
+ font-weight: 700;
+ color: #00a9ce;
+ box-shadow: 0 1px 0 1px rgba(0,0,0,.04);
+ border: 2px solid #999;
+ -moz-appearance: none;
+ -webkit-appearance: none;
+ appearance: none;
+ background-color: #fff;
+ background-image: url('../images/dropdown.svg');
+ background-repeat: no-repeat, repeat;
+ background-position: right .7em top 50%, 0 0;
```

Comment:
I am not sure about this syntax but I would think that everything after the comma does not need to be there.